app/src/ui/app.tsx

```diff
@@ -1363,8 +1364,11 @@ export class App extends React.Component<IAppProps, IAppState> {
    * on Windows.
    */
   private renderAppMenuBar() {
-    // We only render the app menu bar on Windows
-    if (!__WIN32__) {
+    // We render the app menu bar on Windows and Linux
+    if (
+      __DARWIN__ ||
+      (__LINUX__ && this.props.appStore.getState().titleBarStyle === 'native')
```

Comment:
:art: we can simplify this because the `IAppState` that you're retrieving here is actually directly accessible:

```suggestion
      (__LINUX__ && this.state.titleBarStyle === 'native')
```